### DISCUSSION TITLE
ER_TOO_MANY_KEYS bug?

### DISCUSSION THREAD

**Discussion by AntennaeVY** [sequelize/sequelize#13147] [Help & Questions]
I've got a MySQL database setted up with Sequelize and `sync({ alter: true })`, this means everytime I restart the server it's going to `ALTER TABLE` multiple times, what happens is that if you use nodemon, at least in development mode it's going to restart and alter your table everytime you save your files, and the only way I've found to fix this issue is to `DROP TABLE` everytime this error appears lol _(every 64 restarts)_, should this be considered as a bug? 

Whole error: 

```
code: 'ER_TOO_MANY_KEYS',
    errno: 1069,
    sqlState: '42000',
    sqlMessage: 'Too many keys specified; max 64 keys allowed',
    sql: 'ALTER TABLE `Users` CHANGE `username` `username` VARCHAR(255) NOT NULL UNIQUE;',
    parameters: undefined
  },
  original: Error: Too many keys specified; max 64 keys allowed

```

**Comment by KaliLinuxF** [comment]
same

**Comment by abdrhxyii** [comment]
I got the same error tooo.... comment this ->  sync({ alter: true })

**Comment by zgunz42** [comment]
@AntennaeVY I see on my MySQL table index the sequelize try to create a new index not alter existing one cause MySQL throw ER_TOO_MANY_KEYS. I found every run create new index with suffix number for example email then email_1, email_2 and so on for every run
